src/main/java/frc/robot/autoCommands/autoTurn.java

```diff
@@ -35,22 +35,22 @@ public autoTurn(DrivetrainInterface drivetrain, double rotation, double maxVeloc
 
   public autoTurn(DrivetrainInterface drivetrain, double rotation) {
 
-    this(drivetrain, rotation, Constants.MAX_VELOCITY);
+    this(drivetrain, rotation, Constants.MAX_DRIVETRAIN_VELOCITY);
   }
 
-   public static autoTurn auto135left(DrivetrainInterface drivetrain) {
+  public static autoTurn auto135left(DrivetrainInterface drivetrain) {
     return new autoTurn(drivetrain, -135);
   }
   
   public static autoTurn auto90left(DrivetrainInterface drivetrain) {
     return new autoTurn(drivetrain, -90);
   }
 
-    public static autoTurn auto45left(DrivetrainInterface drivetrain) {
+  public static autoTurn auto45left(DrivetrainInterface drivetrain) {
```

Comment:
what about a single autoLeft and pass the angle in ... and autoRight and pass the angle in?